libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -145,9 +162,32 @@ void AC_PrecLand::init()
     }
 }
 
+// create inertial buffer
+void AC_PrecLand::check_inertial_buffer()
+{
+    // if lag parameter is not within bounds, return without doing anything
+    if (_lag_ms <= 0 || _lag_ms > 250) {
+        return;
+    }
+
+    // calculate inertial buffer size from delay length (loop runs at 400hz)
```

Comment:
Rather then hardcoding the loop rate is there any chance of making this use the Scheduler loop rate instead, or as an optional input parameter which we take the min of vs the loop rate? The main reason I ask is people have asked about doing this on plane, but planes don't typically run at 400Hz so this creeps more difficulties in to bringing the feature to other vehicles (and potentially to copter when the loop rates are speed up).

Reply:
sure, we can use AP::scheduler().get_loop_rate_hz().  I'll add this.